src/schema/schema.ts

```diff
@@ -78,6 +78,8 @@ export function createZimbraSchema(
 		notificationHandler: notifications && notifications.notificationHandler
 	});
 
+	const localStoreClient = options.localStoreClient;
```

Comment:
How will this look when we cut over to a real GraphQL backend? I think we may have to move this to the `@client` resolvers.